R/reorder.R

```diff
@@ -97,3 +103,20 @@ fct_infreq <- function(f, ordered = NA) {
 
   lvls_reorder(f, order(table(f), decreasing = TRUE), ordered = ordered)
 }
+
+
+#' @export
+#' @rdname fct_inorder
+fct_inseq <- function(f, desc = FALSE) {
```

Comment:
Should this have `ordered = NA` to match the other functions in this family?

This is also the only function to use `desc`, which is surprising. Maybe remove for now?